brazil_flag.c

```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
-#include <conio.h>
+#include <ncurses.h> // LINUX
```

Comment:
You could do something like this to support both 😄

```c
#ifdef __linux__ 
  #include <ncurses.h>
#elif _WIN32
  #include <conio.h>
#endif
```

[ref](https://stackoverflow.com/questions/6649936/c-compiling-on-windows-and-linux-ifdef-switch)

Reply:
Script updated.

Reply:
nice!